.gitignore

```diff
@@ -20,9 +20,12 @@ thumbnail.bmp
 .idea
 xcshareddata
 
-# Python distutils
-/MANIFEST
```

Comment:
We could also leave `MANIFEST` in there, or not?

Reply:
Now that setuptools is used instead of distutils  this file is not generated anymore. Instead it is generated in `musicplayer.egg-info/SOURCES.txt`.

But you are right as previously this file could have been generated I should leave it there.

Reply:
I added it back in 5ec0a066